quint/src/itf.ts

```diff
@@ -126,6 +173,60 @@ export function toItf(vars: string[], states: QuintEx[]): Either<string, ItfTrac
   })
 }
 
-// export function ofItf(itf: any[]): Either<string, QuintEx[]> {
-//   const stateToExpr = (any: )
-// }
+export function ofItf(itf: ItfTrace): QuintEx[] {
+  // Benign state to synthesize ids for the Quint expressions
+  var nextId = 0n
```

Comment:
```suggestion
  let nextId = 0n
```

Reply:
Oh yes. Thanks. I forgot JS has two different mutable variables, one of which is not to be used lol. Thanks.